InnerEye/ML/configs/histo_configs/classification/DeepSMILECrck.py

```diff
@@ -129,7 +129,9 @@ def get_data_module(self) -> TilesDataModule:
 
     def get_trainer_arguments(self) -> Dict[str, Any]:
         # These arguments will be passed through to the Lightning trainer.
-        return {"callbacks": self.callbacks}
+        kw_args = super().get_trainer_arguments()
+        kw_args["callbacks"] = self.callbacks
+        return kw_args
```

Comment:
LightningContainer now has a get_callbacks method to simplify that.

Reply:
Done, thanks.